contracts/ArenaGovernor.sol

```diff
@@ -3,31 +3,32 @@ pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts/governance/Governor.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
-import "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol";
+import "@openzeppelin/contracts/governance/compatibility/GovernorCompatibilityBravo.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
-import "@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol";
 import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
 
 contract ArenaGovernor is
     Governor,
     GovernorSettings,
-    GovernorCountingSimple,
+    GovernorCompatibilityBravo,
     GovernorVotes,
-    GovernorVotesQuorumFraction,
     GovernorTimelockControl
 {
     constructor(ERC20Votes _token, TimelockController _timelock)
-        Governor("CodeArenaGovernor")
+        Governor("ArenaGovernor")
         GovernorSettings(
             1, /* 1 block */
             302400, /* 1 week */
```

Comment:
The comment should be updated (1 week == 45818)
Note: after the merge the blocktime will 12 seconds; probably doesn't make much of a difference

Reply:
oh, this will be deployed on Polygon; block time is between 2 to 2.5s. Using the OZ wizard generator with 2s resulted in this value.

Reply:
oh yes of course....  (might add a comment about polygon here to guide other readers)